bower.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "cfgov-sheer-templates",
   "description": "Sheer templates for consumerfinance.gov",
-  "version": "0.3.1",
+  "version": "1.0.0",
```

Comment:
`version` is deprecated. Not a big deal that it's there because it's ignored by bower but it should be removed at some point.